roles/neutron/defaults/main.yml

```diff
@@ -27,3 +27,4 @@ neutron_ingress_annotations: {}
 
 # Enable dns integration
 neutron_designate_integration_enabled: false
+neutron_policy_server_integration_enabled: false
```

Comment:
Honestly, I see the usefulness of this for everyone, so let's enable it by default, and add a comment above it and an extra space.